packages/firebase_remote_config/firebase_remote_config_platform_interface/lib/src/enums.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+part of firebase_remote_config_platform_interface;
+
+/// ValueSource defines the possible sources of a config parameter value.
+enum ValueSource {
```

Comment:
Nit: Could we add this to the `firebase_remote_config_platform_interface.dart` file?

Reply:
Sure, will do